app/src/test/java/com/github/multimatum_team/multimatum/QRGeneratorActivityTest.kt

```diff
@@ -61,13 +65,15 @@ class QRGeneratorActivityTest {
 
     @Test
     fun qRDisplayTest() {
+        val data = Deadline("Appeller Robert", DeadlineState.TODO, LocalDateTime.now()
+            .plusDays(1))
         val intent = Intent(
```

Comment:
you must not use localDateTime.now() but use mocked clockService instead or localdateTime.of(...) if time doesnt matter